magprime/automated_emails.py

```diff
@@ -34,28 +34,28 @@ def __init__(self, event):
                sender='MAGFest LAN <lan@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Tech Ops volunteering', 'techops.txt',
-           lambda a: a.staffing and a.requested(c.TECH_OPS) and not a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>')
+           lambda a: a.staffing and a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>')
```

Comment:
Is this change intentional?  Previously this email was for people who requested tech ops but who had NOT yet been assigned there.  Now it's being sent to people who have been assigned to tech ops.